lib/simulator-xcode-9.js

```diff
@@ -600,6 +601,8 @@ class SimulatorXcode9 extends SimulatorXcode8 {
           `The 'AppleLanguages' preference is already set to '${globalPrefs.AppleLanguages}'. ` +
           `Skipping services reset`
         );
+      } else if (language?.skipSyncUiDialogTranslation) {
+        log.info('Skipping services reset because skipSyncUiDialogTranslation is true');
```

Comment:
Skipping services reset as requested. This might leave some system UI alerts untranslated

Reply:
Thanks! Updated as https://github.com/appium/appium-ios-simulator/pull/404/commits/a29b08df479fb3c5f55c9c33bd7f6102a73c580d